requirements/developer.txt

```diff
@@ -4,8 +4,8 @@ pytest
 pylint==2.6.0
 pytest-mock
 pytest-cov
-nvgpu
-numpy
+nvgpu; sys_platform != 'win32'
+nvgpu==0.8.0; sys_platform == 'win32'
 grpcio
 protobuf
 grpcio-tools
```

Comment:
should transformer be set as 4.0.0?

Reply:
@lxning: Transformers is a very BERT model-specific package. This PR targets to stabilize the install_dependency script for different OS/CUDA environment.

Upgrading the transformers version to 4.0.0 will add un-necessary testing and fixes of BERT models. This should be taken up as part of #927